src/App.tsx

```diff
@@ -21,14 +77,18 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+              <TodoList
+                errorMessage={errorMessage}
+                todos={visibleTodos}
+                // selectTodo={selectTodo}
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)